app/controllers/facility_statements_controller.rb

```diff
@@ -38,9 +38,9 @@ def create
       flash[:error] = text("no_selection")
     elsif @statement_creator.create
       @statement_creator.send_statement_emails
-      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list).html_safe
+      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list.html_safe).html_safe
```

Comment:
...I started to write out why I think we still need it and then realized I agree, we don't. Will take it out. 😸